package.json

```diff
@@ -22,8 +22,12 @@
     "docsify": "bin/docsify"
   },
   "scripts": {
-  	"test": "ava e2e/index.js",
-    "release": "standard-version"
+    "pretest": "npm run build",
```

Comment:
no need of `pretest` just add build scripts in the travis

Reply:
I was wondering where `test` is specified.

I mean, I can add it on `.travis.yml` but in the current master version isn't there:

https://github.com/docsifyjs/docsify-cli/blob/master/.travis.yml

🤔

Reply:
travis automatically looks for `test` script.

Reply:
I see.

Well, I changed it but it's still failing